pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -436,7 +436,7 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 		}
 
 		if cr.Spec.Unmanaged && !replset.Expose.Enabled {
-			return errors.Errorf("replset %s needs to be exposed if cluster is unmanaged", replset.Name)
+			log.Info("consider exposing replset for unmanaged clusters", "replset", replset.Name)
```

Comment:
I feel like something like this may better explain the purpose of the log:
```suggestion
			log.Info("Replset is not exposed. Make sure each pod in the replset can reach each other.", "replset", replset.Name)
```

Reply:
Good suggestion.